tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "outDir": "lib",
+    "allowJs": true,
```

Comment:
Is this only temporary while wip?

Reply:
Are you referring to `allowJs`? If so, then yes. It's for WIP so that TS compiler copies over `*.js` files into `lib` for testing.